src/components/CardProduct.js

```diff
@@ -19,12 +27,13 @@ class CardProduct extends Component {
           <img src={ thumbnail } alt={ title } />
           <p>{ price }</p>
         </Link>
+        { freeShipping ? <FreeShipping /> : null }
         <button
           data-testid="product-add-to-cart"
           onClick={ () => addToCartFunction(product) }
           type="button"
         >
-          Adiiconar ao carrinho
+          Adiconar ao carrinho
```

Comment:
```suggestion
          Adicionar ao carrinho
```
:thinking: Acho que ta errado aqui

Reply:
Algo de errado nao esta certo 😭